### ISSUE TITLE
New collection funder image

### ISSUE THREAD

**Issue by apinkney0696** [sciencehistory/scihist_digicoll#1966]
Hi! Could I please have the following image added to the dropdown list for the Funder Image field in Collection-level metadata:

![National-Science-Foundation-logo](https://user-images.githubusercontent.com/78036563/204308681-2150b6ea-8f25-4e80-942b-f4de8049739e.jpg)

**Comment by jrochkind** [comment]
Yup. @apinkney0696  what do you think about how these images are looking? 

I feel like I designed the page thinking most of them would have solid color backgrounds or be square, since those were the examples I had at the time, but most of them, like this, have ended up being non-square, floating on.a white background. And to me it ends up looking kind of busy/confusing. What do you think, is it worth trying to clean this up?

I'm not sure if that might mean redoing some of these images to have *transparent* backgrounds.  Honestly that might be a good practice anyway to be future proof -- unless they are actually intended to be on white square specifically, like that's part of the branding/logo or whatever. Any thoughts? 

What format did you try to attach? It looks like it might be a JPG?  I wonder if it might be better to use PNG generally.

**Comment by apinkney0696** [comment]
I think those currently online look okay so far! I didn't edit this image at all and just put it straight into the ticket from my internet download. I forgot that for the others I had edited them to make them square in advance of sending them to you. I will absolutely make this square and a transparent .png before uploading another copy - a good practice to establish, just so they're all clean and consistent!

**Comment by jrochkind** [comment]
Cool cool if i'm the only one that thinks it looks messy as it is, I'll just try to ignore it longer. :) 

But yeah, making sure it's exact square is good, at minimum. Thanks!
